components/feature/downloads/src/test/java/mozilla/components/feature/downloads/AbstractFetchDownloadServiceTest.kt

```diff
@@ -1074,14 +1074,17 @@ class AbstractFetchDownloadServiceTest {
         service.setDownloadJobStatus(downloadJobState, DownloadState.Status.PAUSED)
         assertEquals(DownloadState.Status.PAUSED, service.getDownloadJobStatus(downloadJobState))
 
-        testDispatcher.advanceTimeBy(PROGRESS_UPDATE_INTERVAL)
+        mainDispatcher.scheduler.advanceTimeBy(PROGRESS_UPDATE_INTERVAL)
+        mainDispatcher.scheduler.runCurrent()
 
         // one of the notifications it is the group notification only for devices the support it
         assertEquals(2, shadowNotificationService.size())
     }
 
     @Test
     fun `notification is shown when download status is COMPLETED`() = runBlocking {
+        // val dispatcher = UnconfinedTestDispatcher()
+        // Dispatchers.setMain(dispatcher)
```

Comment:
nit: leftover?

Reply:
Seems like, will remove this, thank you!